bedrock/settings/base.py

```diff
@@ -1748,12 +1750,372 @@ def before_send(event, hint):
+    },
+    # Finland
+    "FI": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["euro"]["fi"],
+    },
+    # USA
+    "US": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["en"],
+    },
+    # Great Britian
+    "GB": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    # Canada
+    "CA": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["en"],
+    },
+    # New Zealand
+    "NZ": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    # Malaysia
+    "MY": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+    # Singapore
+    "SG": {
+        "default": RELAY_EMAIL_PLAN_ID_MATRIX["usd"]["gb"],
+    },
+}
+
+# Countries where Relay Email Masking is available.
+RELAY_EMAIL_COUNTRY_CODES = [
+    "CA",  # Canada
+    "MY",  # Malaysia
+    "NZ",  # New Zealand
+    "SG",  # Singapore
+    # United Kingdom + "Territories"
+    "GB",  # United Kingdom of Great Britain and Northern Island
+    "GG",  # Guernsey (a British Crown dependency)
+    "IM",  # Isle of Man (a British Crown dependency)
+    "IO",  # British Indian Ocean Territory
+    "JE",  # Jersey (a British Crown dependency)
+    "UK",  # United Kingdom
+    "VG",  # Virgin Islands (British)
+    # USA + "Territories"
+    "AS",  # American Samoa
+    "MP",  # Northern Mariana Islands
+    "PR",  # Puerto Rico
+    "UM",  # United States Minor Outlying Islands
+    "US",  # United States of America
+    "VI",  # Virgin Islands (U.S.)
+    # EU Countries
+    "AT",  # Austria
+    "BE",  # Belgium
+    "CH",  # Switzerland
+    "DE",  # Germany
+    "ES",  # Spain
+    "FI",  # Finland
+    "FR",  # France
+    "IE",  # Ireland
+    "IT",  # Italy
+    "NL",  # Netherlands
+    "SE",  # Sweden
+]
+
+RELAY_PHONE_PLAN_ID_MATRIX = {
+    "usd": {
+        "en": {
+            "monthly": {
+                "id": "TKTK" if DEV else "price_1Li0w8JNcmPzuWtR2rGU80P3",
+                "price": 4.99,
+                "currency": "USD",
+            },
+            "yearly": {
+                "id": "TKTK" if DEV else "price_1Li15WJNcmPzuWtRIh0F4VwP",
+                "price": 3.99,
+                "currency": "USD",
+            },
+        },
+    },
+}
+
+RELAY_PHONE_PLAN_COUNTRY_LANG_MAPPING = {
+    "US": {
+        "default": RELAY_PHONE_PLAN_ID_MATRIX["usd"]["en"],
+    },
+    "CA": {
+        "default": RELAY_PHONE_PLAN_ID_MATRIX["usd"]["en"],
+    },
+}
+
+# Countries where Relay Phone Masking is available.
+RELAY_PHONE_COUNTRY_CODES = [
+    "CA",  # Canada
+    "US",  # United States of America
+]
+
+
+# RELAY / VPN BUNDLE ============================================================================
+
+# Bundle guarantee period
+RELAY_VPN_BUNDLE_GUARANTEE = 30
+
+# Product ID for VPN & Relay bundle subscriptions.
+RELAY_VPN_BUNDLE_PRODUCT_ID = config("VPN_RELAY_BUNDLE_PRODUCT_ID", default="prod_MQ9Zf1cyI81XS2" if DEV else "prod_MIex7Q079igFZJ")
```

Comment:
I ended up duplicating all the VPN bundle stuff even though it was still there from when the bundle was on the VPN pages. I needed to display the pricing stuff in a format that it wouldn't handle. But I don't like the duplication. Can I delete the VPN one?